vagrant/puppet/cmf/manifests/application/php.pp

```diff
@@ -0,0 +1,56 @@
+class cmf::application::php(
+  $version     = '5.5',
+  $use_apc     = undef,
+  $use_xdebug  = undef,
+  $environment = hiera('environment', 'dev')
+) {
+  validate_string($environment)
+
+  case $version {
+    '5.4':   { $ppaVersion = 'php5-oldstable' }
```

Comment:
i would say only provide maintained versions of php, so 5.5+

5.3 is long dead, and 5.4 has reached end of live as well.